docs/pages/linking/share-content-into-your-app.mdx

```diff
@@ -0,0 +1,122 @@
+
+You may want to let other apps share content into your app &mdash; like articles, images, or files &mdash; using the native system share sheet. This guide explains how that works under the hood and gives an overview of the tools available to help you implement it.
+
+With [`expo-sharing`](/versions/latest/sdk/sharing), you can already share content _from_ your app into other apps. Receiving content works the other way around: your app needs to become a share target, which on Android means responding to Intents and on iOS means handling a Share Extension.
+
+## OS foundations
+
+**Android**
+
+On Android, the system delivers shared data to an Activity. An Activity is a single, focused screen in your app, and it can also act as an entry point that the system launches directly when the user selects your app in the share sheet.
+
+- Apps register intent filters (ACTION_SEND / ACTION_SEND_MULTIPLE) for specific MIME types (e.g., text/plain, image/\*) in AndroidManifest.xml.
+- When selected from the share sheet, your launch Activity receives an Intent containing the data payload.
+- Your Activity must handle the incoming Intent in Java or Kotlin by calling getIntent() and reading extras like EXTRA_TEXT or EXTRA_STREAM.
+
+**iOS**
+
+On iOS, receiving shared data is handled by a Share Extension that runs separately from your main app.
+
+- A Share Extension lets users send content into your app.
+- The system launches your extension with an NSExtensionContext containing NSExtensionItem attachments.
+- The extension processes those items (e.g., reads text/URLs/files). To make them available to the main app, a common approach is to write data to a shared App Group container and then open the app (via a custom URL or universal link).
+- A Share Extension is implemented as its own target. It runs as a separate process from your main app and is written in Swift or Objective-C, since it needs to use iOS system APIs to receive shared content.
+
+For more in-depth details, see the official platform documentation:
+
+- Android: [Receiving simple data from other apps](https://developer.android.com/training/sharing/receive)
+- iOS: [App Extension Programming Guide – Share](https://developer.apple.com/library/archive/documentation/General/Conceptual/ExtensibilityPG/Share.html)
+
+## Available libraries
+
+Expo does not currently provide its own official library for receiving shared content. Instead, you'll need to rely on community-maintained packages or build a custom solution. Here are some options that Expo developers commonly use:
+
+**Expo Share Intent**
+
+- Cross-platform support (Android and iOS) for receiving shared data.
+- Adds a Share Extension on iOS and intent support on Android.
+- Works in Expo with a development build and exposes a simple JS hook.
+
+**Expo Apple Targets**
+
+- Enables additional iOS targets such as Share Extensions, widgets, and Siri intents.
+- More powerful and flexible if you're focused on advanced iOS functionality.
```

Comment:
```suggestion
- More powerful and flexible if you're focused on advanced iOS functionality.
```

Let's add a BoxLink here as well to link the library's GitHub repo: https://github.com/EvanBacon/expo-apple-targets.